.circleci/config.yml

```diff
@@ -184,6 +184,39 @@ jobs:
         command: |
           ./bin/clean_up_ecr
 
+  run_webhint:
+    <<: *build_container_config
+    steps:
+    - checkout
+    - *update_packages
+    - *install_clamav
+    - *install_libreoffice
```

Comment:
This could be my ignorance as to how to build CircleCI containers ... but do ClamAV and LibreOffice need to installed here?

Reply:
Yes because to generate the reports we need to run the cucumber tests. And some of them might need to go through the steps requiring LibreOffice and ClamAV